demos/python_demos/3d_segmentation_demo/3d_segmentation_demo.py

```diff
@@ -125,10 +143,10 @@ def bbox3(img):
     return np.array([[-1, -1, -1], [0, 0, 0]])
 
 
-def read_nii_header(data_path, series_name, suffix, separate_folder=True):
-    filename = os.path.join(data_path, series_name, series_name + suffix)
+def read_nii_header(data_path, name, suffix='', separate_folder=True):
```

Comment:
Given that `separate_folder` is always specified as `false`, perhaps it would be better to remove the `suffix` and `separate_folder` parameters entirely? The caller can append a suffix if it needs it.

Reply:
done

Reply:
But the parameters are still there. What did you do?